src/conn/conn_layered.c

```diff
@@ -1159,32 +1135,22 @@ __wti_disagg_check_local_files(WT_SESSION_IMPL *session, const char *cfg[])
     else
         fail = false; /* Default: delete */
 
+    /* Delete from home directory. */
+    WT_RET(__wt_cleanup_start_dir(session, "", fail));
```

Comment:
The naming here doesn't resonate with me. It is either a check or a cleanup depending on the `fail` input parameter. It's really something like "ensure_clean_startup_dir" or something like that.